set.mm

```diff
@@ -82806,6 +82898,7 @@ equivalent to the law of the excluded middle (LEM), and in ILE the LEM
       UAMZVKUBMZWBSZTZSZVJWBTZRZIZFUGZUBUAWEWLUHZUIWDUJVJWMTUIUKZCULWKEMZVJRZWD
       VKWEWOSZTZSZVJWOTZRZIFEWBWONZWCWPWJXAWBWOVJUMXBWHWSWIWTXBWGWRWDXBWFWQVKWB
       WOWEUNUOUPWBWOVJUQURQUSWNUTDULVDVAVBVC $.
+    $( $j usage 'sbth' avoids 'ax-inf' 'ax-inf2' 'ax-ac'; $)
```

Comment:
Interestingly, we have two axioms of choice: [`ax-ac`](https://us.metamath.org/mpeuni/ax-ac.html) and [`ax-ac2`](https://us.metamath.org/mpeuni/ax-ac2.html).
I suppose we need to exclude both everywhere?

Reply:
> Interestingly, we have two axioms of choice: [`ax-ac`](https://us.metamath.org/mpeuni/ax-ac.html) and [`ax-ac2`](https://us.metamath.org/mpeuni/ax-ac2.html). I suppose we need to exclude both everywhere?

Good catch. It seems better to list them both, rather than leave it to convention in terms of "oh this is the one we use" versus "this is the one which is just hanging out there". I'll update this pull request to do that, probably later today.